crates/iota-core/src/authority/consensus_quarantine.rs

```diff
@@ -463,6 +514,8 @@ impl ConsensusOutputQuarantine {
             builder_checkpoint_summary: BTreeMap::new(),
             builder_digest_to_checkpoint: HashMap::new(),
             shared_object_next_versions: RefCountedHashMap::new(),
+            congestion_control_randomness_object_debts: RefCountedHashMap::new(),
+            congestion_control_object_debts: RefCountedHashMap::new(),
```

Comment:
```suggestion
            congestion_control_object_debts: RefCountedHashMap::new(),
            congestion_control_randomness_object_debts: RefCountedHashMap::new(),
```
not a biggie, but my OCD says it needs to be in order :D